firebase-inappmessaging/src/testData/inappmessaging/testutil/TestData.java

```diff
@@ -39,6 +41,10 @@
       new CampaignMetadata(CAMPAIGN_ID_STRING, CAMPAIGN_NAME_STRING, IS_NOT_TEST_MESSAGE);
   public static final CampaignMetadata TEST_CAMPAIGN_METADATA_MODEL =
       new CampaignMetadata(CAMPAIGN_ID_STRING, CAMPAIGN_NAME_STRING, IS_TEST_MESSAGE);
+  public static final Map<String, String> DATA = new HashMap<>();
+  static {
+    DATA.put("up", "dog");
+  }
```

Comment:
Lets use `com.google.common.collect.ImmutableMap` to create an immutable map here

```java
public static final Map<String, String> DATA = ImmutableMap.of("foo", "bar");
```